internal/driver/checkstatuses_test.go

```diff
@@ -36,3 +53,156 @@ func TestUpdateDeviceStatus_noUpdate(t *testing.T) {
 	require.NoError(t, err)
 	assert.False(t, changed)
 }
+
```

Comment:
please remove any leftover commented out code before merging

Reply:
Sounds good, I am still playing around with that, but I will remove it tonight if I'm not getting anywhere.